tigris/client.go

```diff
@@ -65,6 +66,13 @@ func NewClient(ctx context.Context, cfg ...*Config) (*Client, error) {
 		pCfg = *cfg[0]
 	}
 
+	if pCfg.Project == "" {
```

Comment:
IMO this should happen where the rest of the environment parsing is happening otherwise it is not consistent

Reply:
see above